clients/apps/web/src/components/Subscriptions/SubscriptionGroupIcon.tsx

```diff
@@ -36,7 +38,7 @@ const SubscriptionGroupIcon: React.FC<SubscriptionGroupIconProps> = ({
       className={`inline-flex items-center text-[--var-icon-color]`}
       style={style}
     >
-      <IconComponent className="!h-4 !w-4" />
+      <IconComponent className={`!h-${size} !w-${size}`} />
```

Comment:
Sadly, the dynamic types here will not play well with the Tailwind JIT compiler.

It scans the sources at _compile time_, not runtime. It might work out of luck if another component specified the same height/width, but if it uses a size that isn't used anywhere else the class will not be generated.